plan9/pdcplan9.c

```diff
@@ -63,7 +63,10 @@ static void fatal(char *s)
 
 void p9napms(int ms)
 {
-	sleep(ms);
+    /* posix sleep is in seconds... nanosleep did not work
+     alternatives? */
+    
+	sleep(ms/1000);
```

Comment:
Was there anything wrong with calling `_SLEEP(ms)` like I did in my port?

Reply:
Oh... must have missed that, lost it some time ...? in my version it was with sleep().

I now did a version with nanosleep() but indeed if _SLEEP() works in APE that sounds much nicer. I will test it now.

Reply:
_SLEEP(ms) worked perfectly! Really weird that I had sleep() in my copy. I doubt that I would have changed it.

Reply:
Cool, maybe you had a previous copy, I think this one is the latest.
http://9p.io/sources/contrib/fgb/root/sys/src/ape/lib/curses/plan9/

BTW, it's really awesome you've taken the time to update the port.

Reply:
Thanks! Indeed I extracted the relevant sources from a tarball of a ported 3.0, but it was long ago (I think I did the first update ~2015 or so). Because of that I don't remember the details that well.